src/lib/utils/hooks/SynapseAPI/oauth/useOAuthClient.ts

```diff
@@ -78,5 +79,8 @@ export function useMutateOAuthClient(
         await options.onSuccess(updatedClient, variables.client, ctx)
       }
     },
+    onError: (err: any) => {
```

Comment:
This is a good pattern to follow.  Also add the same onError to useDeleteOAuthClient()?